android/src/main/java/com/bitmovin/player/reactnative/BackgroundPlaybackManager.kt

```diff
@@ -52,3 +57,5 @@ class MediaSessionConnectionManager(val context: ReactApplicationContext) {
         playerModule: PlayerModule? = context.playerModule,
     ): Player = playerModule?.getPlayerOrNull(nativeId) ?: throw IllegalArgumentException("Invalid PlayerId $nativeId")
 }
+
+open class PlayerServiceBinder(open var player: Player?) : Binder()
```

Comment:
I am not sure if this is the best way for extending `Binder()` for our use-case. I would like input here from someone more knowledgeable in Kotlin 🙏